explainer.md

```diff
@@ -607,14 +607,24 @@ enum XREye {
   "right"
 };
 
+[SecureContext, Exposed=Window,
+ Constructor(optional DOMPointInit translation, optional DOMPointInit orientation)]
+interface XRRigidTransform {
+  readonly attribute DOMPointReadOnly translation;
+  readonly attribute DOMPointReadOnly orientation;
+
+  readonly attribute Float32Array matrix;
+};
+
 [SecureContext, Exposed=Window] interface XRView {
   readonly attribute XREye eye;
   readonly attribute Float32Array projectionMatrix;
   readonly attribute Float32Array viewMatrix;
+  readonly attribute XRRigidTransform transform;
```

Comment:
Is `transform.matrix` always the same as `viewMatrix` here?

Reply:
If so, perhaps we just combine these two attributes as `viewTransform`?

Reply:
No, that was not my intent (because otherwise it's just data duplication.) The transform here would not be inverted, and therefore the matrix wouldn't be either. So that makes it effectively the model matrix for the camera, and the view matrix that sits outside the interface would be the inverted version of the matrix.

This definitely needs to be explained better.

Reply:
Updated some of the text around this (and ended up fleshing out some underexplained XRView concepts in the process.) Thanks for calling it out!